adapter/src/main/java/com/twilio/kudu/sql/KuduQuery.java

```diff
@@ -64,6 +65,10 @@ public KuduQuery(RelOptCluster cluster, RelTraitSet traitSet, RelOptTable table,
     this.calciteKuduTable = calciteKuduTable;
     this.projectRowType = projectRowType;
     assert getConvention() == KuduRelNode.CONVENTION;
+    // include our own metadata provider so that we can customize costs
+    JaninoRelMetadataProvider relMetadataProvider = JaninoRelMetadataProvider.of(KuduRelMetadataProvider.INSTANCE);
```

Comment:
Need to use our own metadata provider so this has to be set in the constructor of all kudu rel nodes.